components/Slider/src/MDCSlider.m

```diff
@@ -121,6 +123,8 @@ - (void)commonMDCSliderInit {
     _hapticsEnabled = NO;
   }
   _shouldEnableHapticsForAllDiscreteValues = NO;
+
+  _firstValueChange = true;
```

Comment:
_Tip:_ As a convention we typically use `YES | NO` within our components instead of `true | false`